src/middlewares/authenticationMiddleware.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require("jsonwebtoken");
+
+// Middleware to check authentication status
+exports.isAuthenticated = (req, res, next) => {
+  const token = req.cookies.jwtToken;
```

Comment:
check the token from `req.header('Authorization')` as well because once you use the frontend, the frontend will pass the token through headers as well